packages/stream/get-stream/get-stream.test.js

```diff
@@ -0,0 +1,132 @@
+import nock from 'nock'
+import { expect } from 'chai'
+import { getStream } from './get-stream.js'
+import { Internal } from '../../internal/index.js'
+import { init } from '../../app/init/init.js'
+
+describe('Get Stream Module', function () {
+  describe('Basic test', function () {
+    it('should be a function', function () {
+      expect(getStream).to.be.a('function')
+    })
+  })
+
+  describe('Negative test', function () {
+    it('should return error response if function is called with no stream ID argument', async function () {
+      try {
+        await getStream()
+      } catch (error) {
```

Comment:
We don't need to catch the error here too

Reply:
resolve as per above [notes](https://github.com/inlivedev/inlive-js-sdk/pull/12#discussion_r973874924)